bpf/lib/tunnel.h

```diff
@@ -5,6 +5,11 @@
 #define __LIB_TUNNEL_H_
 
 #include <linux/ipv6.h>
+#include <linux/in.h>
+#include <linux/ip.h>
+#include <linux/udp.h>
+#include <linux/if_ether.h>
+#include "lib/csum.h"
```

Comment:
Just to note - as mentioned offline, I had forked off `tunnel.h` to be the "simple" header that just contains definitions and trivial helpers. Your additions *might* be better placed in `encap.h` for the time being. But no biggie.

Reply:
Hm, I'm definitely not opposed to moving the vxlan helper functions somewhere else? Maybe their own header file? I think `lib/csum/h` is pretty light weight right now. 

While just a note, if we want to handle this better, maybe a new file for the vxlan helper functions is warranted?

Reply:
Fixed in latest push, resolving

Reply:
Ah! My bad I thought we were looking at vxlan.h 😅. 

Yup, ill remove these!

Reply:
Okay, backed out any touches to tunnel.h, tunnel.h should indeed have no diff at all in this PR.